### ISSUE TITLE
unable to asbuild report

### ISSUE THREAD

**Issue by kedargodabole** [AsBuiltReport/AsBuiltReport.VMware.vSphere#125]
### Bug description

i am running as build report from windows server and my server doesnt have internet. Hence i manually downloaded latest powercli and asbuildreport.vmware.vsphere module and pasted the files in c:\windows\system32\windowspowershell\1.0\modules folder.
 then i run the command to create json file :New-AsBuiltReportConfig -Report 'VMware.vSphere' -Folderpath 'C:\TEMP' -Force
and getting below error. not sure what i am missing. I am not good at PS scripting just trying the script.
New-AsBuiltReportConfig : The term 'New-AsBuiltReportConfig' is not recognized as the name of a cmdlet, function, script file,

### Command-line input

New-AsBuiltReportConfig -Report 'VMware.vSphere' -Folderpath 'C:\TEMP' -Force

### Steps to reproduce

explained above

### Expected behaviour

NA

### Operating System

windows server 2016

### PowerShell Version

5.1.14393.7426

### PowerShell Modules

asbuildreport.vmware.vsphere

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://www.asbuiltreport.com/about/contributing/#reporting-issues-and-bugs).
- [X] I have read [the documentation](https://www.asbuiltreport.com/user-guide/new-asbuiltconfig), and referred to the [known issues](https://www.asbuiltreport.com/user-guide/known-issues/) before submitting this bug report.
- [X] I have checked for previously opened & closed [issues](https://github.com/AsBuiltReport/AsBuiltReport.VMware.vSphere/issues) before submitting this bug report.

**Comment by tpcarman** [comment]
Have you followed the [offline installation instructions](https://www.asbuiltreport.com/user-guide/installation/#offline-installation) correctly?

**Comment by kedargodabole** [comment]
yes.. thanks for suggestion. HTML and word documents are getting generated but it has minimal information and script throwing this error. I dont see any issues when i use connectvi-server 'VC-name'
Get-PatchBaseline : 1/11/2024 1:07:45 AM        Get-PatchBaseline               There was no endpoint listening at https://XXXXXXXXXXX/vci/sdk that could accept the message. This is often caused by an incorrect address or SOAP action. See InnerException, if present, for more details.

**Comment by tpcarman** [comment]
The command you have outlined above `New-AsBuiltReportConfig -Report 'VMware.vSphere' -Folderpath 'C:\TEMP' -Force` is used to generate the JSON report configuration file. This is not the command which is needed to generate a report. Please refer to the [New-AsBuiltReport](https://www.asbuiltreport.com/user-guide/new-asbuiltreport/) documentation for guidance on how to generate a report.
